src/compute-client/src/controller.rs

```diff
@@ -608,6 +604,14 @@ where
 
         None
     }
+
+    async fn record_introspection_updates(&mut self) {
+        for (type_, updates) in self.compute.introspection_rx.try_iter() {
+            self.storage
+                .record_introspection_updates(type_, updates)
```

Comment:
This is the first instance of the compute controller interacting with its reference to the storage controller for anything other than updating frontiers, so a bit of a departure

personally i think its fine, but moving the channel underneath to an unbounded one makes me very nervous, so ideally we would either accept the latency cost here (unknown, and depends on load/number of replicas/etc im sure), or we can come up with some other way of dealing with it